schema.sql

```diff
@@ -4,14 +4,35 @@ CREATE DATABASE vet_clinic;
 
 \c vet_clinic;
 
-CREATE TABLE IF NOT EXISTS animals(
-  id              int GENERATED ALWAYS AS IDENTITY, 
-  name            varchar(100) not null,
-  date_of_birth   date not null,
-  escape_attempts int not null,
-  neutered        boolean not null,
-  weight_kg       float not null,
+CREATE TABLE IF NOT EXISTS owners (
+  id        INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(255) NOT NULL,
+  age       INT NOT NULL,
   PRIMARY KEY(id)
 );
 
-ALTER TABLE animals ADD species varchar(40); 
+CREATE TABLE IF NOT EXISTS species (
+  id   INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(40) NOT NULL,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE IF NOT EXISTS animals (
+  id              INT GENERATED ALWAYS AS IDENTITY, 
+  name            VARCHAR(100) NOT NULL,
+  species_id      INT,
```

Comment:
- [ ] all animals have to be assigned owners. This can be achieved by using an update statement, kindly fix this
- Example
![image](https://user-images.githubusercontent.com/81932478/184390787-385c299d-53df-4b6b-9bfd-eabeab4c5f83.png)

- The requirements
![image](https://user-images.githubusercontent.com/81932478/184391199-70ef308a-d5ea-4f65-9485-6e8f3906ec9a.png)
#### Note that the statement used is the `Update` statement.